components/sp-solutions/org.wso2.carbon.sp.solutions.apim.analytics/widgets/APIMApiCreatorAlertConfiguration/src/APIMApiCreatorAlertConfiguration.jsx

```diff
@@ -0,0 +1,205 @@
+                        },
+                        {
+                            "name": "apiCreator",
+                            "title": "Api Creator"
+                        },
+                        {
+                            "name": "apiCreatorTenantDomain",
+                            "title": "TenantDomain"
+                        },
+                        {
+                            "name": "thresholdResponseTime",
+                            "title": "thresholdResponseTime"
+                        },
+                        {
+                            "name": "thresholdBackendTime",
+                            "title": "thresholdBackendTime"
+                        }
+                    ]
+                }
+            ],
+            pagination: true,
+            filterable: true,
+            append: false
+        };
+
+        this.metadata = {
+            "names": [
+                "apiName",
+                "apiVersion",
+                "apiCreator",
+                "apiCreatorTenantDomain",
+                "thresholdResponseTime",
+                "thresholdBackendTime"
+            ],
+            "types": [
+                "ordinal",
+                "ordinal",
+                "ordinal",
+                "ordinal",
+                "ordinal",
+                "ordinal"
+            ]
+        };
+         
+        this.state = {
+            width: this.props.glContainer.width,
+            height: this.props.glContainer.height,
+            data:[],
+            dataProviderConf:[]
+        };
+        
+        this.props.glContainer.on('resize', () =>
+            this.setState({
+                width: this.props.glContainer.width,
+                height: this.props.glContainer.height
+            })
+        );
+
+        this.handleDataReceived = this.handleDataReceived.bind(this);
+        this.handleDataManaged = this.handleDataManaged.bind(this);
+        this.handleSubmit = this.handleSubmit.bind(this);
+        this.handleChange = this.handleChange.bind(this);
+    }
+    
+    componentDidMount() {
+        super.getWidgetConfiguration(this.props.widgetID)
+            .then((message) => {
+                super.getWidgetChannelManager().subscribeWidget(this.props.id,this.handleDataReceived, message.data.configs.providerConfig);
+                console.info(message.data.configs.providerConfig);
+                this.setState({
+                    dataProviderConf : message.data.configs.providerConfig
+                }); 
+            })
+            .catch((error) => {
+                this.setState({
+                    faultyProviderConf: true
+                });
+            }) 
+    }
+
+    componentWillUnmount() {
+        super.getWidgetChannelManager().unsubscribeWidget(this.props.id);
+    }
+
+    handleDataReceived(message) {
+        this.setState({
+            data:message.data
+        });
+    }
+
+    handleDataManaged(message){
```

Comment:
remove method is unnecessary

Reply:
This was sample code added for revision and removed in 45ea197bf4dd881b1e9d79f5b3ba8d3cc12e05c9